dev/bots/analyze.dart

```diff
@@ -584,6 +587,44 @@ Future<void> verifyGoldenTags(String workingDirectory, { int minimumMatches = 20
   }
 }
 
+/// Use of Stopwatches can introduce test flakes as the logical time of a
+/// stopwatch can fall out of sync with the mocked time of FakeAsync in testing.
+/// The Clock object provides a safe stopwatch instead, which is paired with the
+/// FakeAsync as part of the test binding.
+final RegExp _findStopwatchPattern = RegExp(r'Stopwatch\(\)');
+const String _ignoreStopwatch = '// flutter_ignore: stopwatch (see analyze.dart)';
+const String _ignoreStopwatchForFile = '// flutter_ignore_for_file: stopwatch (see analyze.dart)';
```

Comment:
Are we anticipating any legit reasons for opting out of this lint?

Reply:
Not sure yet, the analyzer just found a bunch for me to check out though!

Reply:
I think that the blinking cursor uses a periodic Timer (or it should) since the tick rate is only once every hundred or so ms and the timer avoids pumping empty frames.